docs/source/DebuggingWithGDB.rst

```diff
@@ -185,3 +181,38 @@ breakpoints, executing commands and querying for output.
 
 When you check the output (``--show-job-log``) you can see that despite
 declaring the variable as 0, ff is injected and printed instead.
+
+
+rr wrapper
+----------
+
+It uses the ``--wrapper`` avocado functionality to run matching binaries
+inside the specified script. There is an example `rr` script in
+``/usr/share/avocado/wrappers/rr.sh`` which stores the information in
+test's `output_dir/rr` as `$bin_name.$serial_id`.
+
+Let's have a look at an example with malfunctioning qemu. Run the
+malfunctioning qemu inside avocado with wrapper::
+
+    $ avocado run "/usr/local/bin/qemu-system-aarch64 -machine virt" --wrapper /usr/share/examples/wrappers/rr.sh:*qemu*
```

Comment:
Well it's the simplest test just to run a broken binary (any binary, it does not have to be qemu, I chose this one just because I have a broken version available). What test would you propose? We can't use any of the existing ones as they compile the binaries in `self.workdir`, which is then erased.

Using of simple test should work fine with `--wrapper` (and it does).